ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKGroupedStreamTest.java

```diff
@@ -162,9 +163,10 @@ public void shouldBuildStepForAggregate() {
   }
 
   @Test
-  public void shouldBuildStepForAggregateWhereKeyFormatSupportsBothWrappingAndUnwrapping() {
+  public void shouldBuildStepForAggregateWithKeyFormatSerdeFeatures() {
```

Comment:
We're testing two things, right?
- If key wrapping is present, then the key serde feature is inherited
- If key wrapping is not present, then it is populated

Seems like it should be two separate unit tests? Same for the other test file below.